docs/semgrep-supply-chain/upgrade-guidance.md

```diff
@@ -0,0 +1,125 @@
+---
+slug: upgrade-guidance
+title: Upgrade guidance
+hide_title: true
+description: Know if a vulnerable package or dependency can be easily and reliably upgraded to a fixed version.
+tags:
+  - Semgrep Supply Chain
+  - Semgrep AppSec Platform
+---
+
+import ViewDetailsSsc from "/src/components/procedure/_view-details-ssc.md"
+ 
+# Upgrade guidance
+
+Know if you or your developers can safely and reliably update a vulnerable package or dependency to a fixed version. From there, choose to:
+
+- Have Semgrep open a pull request (PR)
+- Create a Jira ticket
+- Set the finding's triage status as **To fix**
+
+Semgrep's dependency upgrade guidance uses AI to analyze if a finding can be **safely upgraded** or if upgrading the package can cause **breaking changes**. Semgrep can also create a PR to upgrade the package.
+
+## Feature maturity
+
+This feature is in **private beta**. To join the beta, reach out to the [Semgrep support team](/support).
+
+## Supported languages and package managers
+
+- **Python** codebases with the following package managers:
+  - `pip`
+  - `pip-tools`
+  - `pipenv`
+  - Poetry
+
+Semgrep supports the following source code managers:
```

Comment:
I suggest omitting this sentence since we don't have one for Python and I think the paragraph break might be sufficient?